papaparse.js

```diff
@@ -355,7 +355,6 @@
 	function ChunkStreamer(config)
 	{
 		this._handle = null;
-		this._paused = false;
```

Comment:
This is a good change - I think we have consensus that this variable is not used. :+1: